integrations/maltiverse.py

```diff
@@ -71,6 +69,15 @@ class Maltiverse:
     def __init__(
         self, endpoint: str = "https://api.maltiverse.com", auth_token: str = None
     ):
+        """
+        Initialize the Maltiverse class.
+
+        Parameters:
+        - endpoint: str, optional (default: "https://api.maltiverse.com")
+            The API endpoint URL.
+        - auth_token: str, optional
+            The authentication token for the API.
```

Comment:
Given the priority of this PR I think we can leave it like this for now, but in order to be consistent with the rest of the code, we should make these changes to the docstrings

```suggestion
        Parameters
        ----------
        endpoint: str, optional (default: "https://api.maltiverse.com")
            The API endpoint URL.
        auth_token: str, optional
            The authentication token for the API.
```